setup.py

```diff
@@ -54,24 +54,24 @@ def main():
     install_requires = [
         'attrs>=17.3.0',
```

Comment:
Maybe == here as well
and also for
 'jsonpatch>1.14',
 'sqlalchemy<1.4.0;
I do not know what strategy you have for managing dependecies